src/orderbooks.md

```diff
@@ -0,0 +1,204 @@
+
+    await delay(100000);
+  });
+}
+
+main()
+  .catch(console.error)
+  .finally(() => process.exit());
+```
+
+:::
+
+These code samples demonstrate various functionalities, from managing storage subscriptions to executing extrinsics.
+
+## Step-by-step instructions
+
+In this section, you will learn how to perform various operations related to order books on the SORA blockchain in `polkadot.js` (or dotapps.io) starting in the `Developer` -> `Extrinsics` tab. Whether you want to create a new order book, delete an existing one, update its details, change the status, place a limit order, or cancel a limit order, this guide will provide you with clear and concise instructions.
+
+::: warning
+
+Please note that during the execution, there may be certain conditions that prevent the execution of specific transactions. In such cases, you will encounter errors with self-explanatory names. For a comprehensive list of all the errors, you can refer to the [codebase](https://github.com/sora-xor/sora2-network/blob/7de5fb694ee3b7ef1b315d13ba61c5eb25e26cb2/pallets/order-book/src/lib.rs#L267).
+
+:::
+
+### Creating an Order Book
+
+This tutorial will walk you through the process of creating a new order book for a specific asset.
+
+1. Choose `orderBook.createOrderbook(dexId, orderBookId)`
+2. Fill the details:
+
+![](./assets/createOrderBook.png)
+_An order book is being created for the XOR-DAI trading pair, with XOR as the base asset. This selection is based on the chosen dexId, which is set to 0._
+
+3. Verify that the Order book was created. Open `Chainstate` calls -> Choose `orderBook.orderBooks` -> Switch off include option switch -> Run the query.
+
+![](./assets/orderBookCreated.png)
+_The provided image showcases the successful creation of an order book for the specified trading pair._
+
+### Deleting an Order Book
+
+In this tutorial, you will discover how to delete an existing order book. We will guide you through the steps to ensure a smooth and secure removal process.
+
+1. Choose `orderBook.deleteOrderbook(dexId, orderBookId)`
+2. Fill the details:
+
+<!-- TODO. add the image -->
+
+3. Check that the Order book was created. Open `Chainstate` calls -> Choose `orderBook.orderBooks` -> Switch off include option switch -> Run the query. A deleted order book should be missing.
+
+### Updating an Order Book
+
+If you need to modify the parameters of an existing order book, this tutorial will demonstrate the steps to update its details. You will learn how to adjust the price step, volume step, or any other relevant parameters.
+
+<!-- TODO. Finish, when implemented -->
+
+### Changing Order Book Status
+
+This tutorial will explain how to change the status of an order book. You will understand the different statuses available and how to transition between them, such as from "Trade" to "PlaceAndCancel" or "OnlyCancel" to "Stop."
+
+<!-- TODO. Finish, when implemented -->
+
+### Placing a Limit Order
```

Comment:
```suggestion
```
This section is not relevant as most users will just use the UI (Polkaswap or mobile) to place orders. If we have a section about placing a limit order, it should explain how to do so through an API, for people to develop their bot or for market makers to integrate to Polkaswap.

@bgodlin